### PR TITLE
Add AcetylCysteine to heal liver damage

### PR DESCRIPTION
**Pull request by JoshAdamPowell**
## About The Pull Request

Adds AcetylCysteine, a new chemical that heals liver damage.

Recipe:
1u part multiver
1u part phenol
1u part sulphuric acid 
makes 3u AcetylCysteine

Slowly heals liver damage, as long as there are no other chemicals present in the patient.

Heals liver damage at 0.25 / tick
metabolises at 0.1u/tick

## Why It's Good For The Game

Currently, the negative side effects of Aiuri are easily dealt with by oculine. This adds AcetylCysteine, which acts as an equvalent chemical to match libital. 
At the moment liver damage can only be dealt with either by replacement or just waiting. Hopefully this chemical can make people more happy to use libital, especially if the latest CobbyChem change gets merged

(Also first PR, be gentle!)

## Changelog
:cl:
add: Added AcetylCysteine, a medicine which heals liver damage as long as the patient has no other chemicals in their body
/:cl:

### CODE REVIEW

**Comment by ExcessiveUseOfCobblestone** [comment]
remove oculine, add organ healing surgeries 😏

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 1]
```diff
@@ -1342,3 +1342,17 @@
 	L.unignore_slowdown(type)
 	L.Dizzy(0)
 	L.Jitter(0)
+
+/datum/reagent/medicine/AcetylCysteine
+	name = "AcetylCysteine"
```
Comment (nicbn) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: I think it'd be better to stylize as `Acetyl-Cysteine` for consistence (and I also find it easier to read). And to use lowercase name in path (`acetylcysteine` or `acetyl_cysteine`) for consistence as well.

**Comment by Detective-Google** [comment]
> remove oculine, add organ healing surgeries

There is an eye healing surgery already....?
It's not really worth doing sense it does a load of brain damage on failure iirc

**Comment by optimumtact** [comment]
I agreew ith cobby, add organ healing surgery instead.

**Comment by JoshAdamPowell** [comment]
I mean no-one will do a liver-healing surgery on anyone ever so I'm cool (please don't nerf oculine)

**Pull request closed without merging**